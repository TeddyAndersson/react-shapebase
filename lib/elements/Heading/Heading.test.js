import React from 'react';
import Heading from './Heading'
import renderer from 'react-test-renderer';

test('should render Heading', () => {
  const component = renderer.create(
    <Heading size={1}>
      Heading size one
    </Heading>
  );
  let tree = component.toJSON();

  expect(tree).toMatchSnapshot();
})
