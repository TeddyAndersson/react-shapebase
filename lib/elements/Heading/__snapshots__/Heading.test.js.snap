// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Heading 1`] = `
<h1
  className="sc-bdVaJa cHoKWU"
>
  Heading size one
</h1>
`;
